assets/theme-supplement.css

```diff
@@ -1632,3 +1632,33 @@ img {
 .sc-field-outline:focus-within {
   outline-color: var(--sc-color-primary);
 }
+
+.sc-testimonial-slideshow {
+  padding: var(--sc-spacing-xlarge) 0;
```

Comment:
Remove padding from parent slideshow because it could prevent correct expanded images.